libs/dh/market-participant/feature-edit-gridarea/src/lib/market-participant-edit-gridarea/dh-market-participant-edit-gridarea.component.html

```diff
@@ -0,0 +1,57 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<ng-container *transloco="let t; read: 'marketParticipant.organization.create'">
+  <ng-container *rxLet="validation$ as validation">
+    <watt-validation-message
+      type="danger"
+      *ngIf="validation"
+      [label]="t('errorToastLabel')"
+      [message]="validation.error"
+    ></watt-validation-message>
+  </ng-container>
+
+  <ng-container *rxLet="isEditing$ as isEditing">
+    <h1 *ngIf="isEditing">{{ t("organizationEditHeader") }}</h1>
+    <h1 *ngIf="!isEditing">{{ t("organizationCreateHeader") }}</h1>
+  </ng-container>
+
+  <!-- Spinner -->
+  <ng-container *rxLet="isLoading$ as isLoading">
+    <div class="spinner-container" *ngIf="isLoading">
+      <watt-spinner></watt-spinner>
+    </div>
+    <ng-container *rxLet="changes$ as changes">
+      <div [hidden]="isLoading">
+        <!-- Tabs -->
+        <watt-tabs class="tabs">
+          <watt-tab [label]="t('masterDataTabHeader')"> </watt-tab>
```

Comment:
This will be part of a follow-up PR, this was just to get the inital grid area list page visible, all other pages will be edited later to represent grid-areas, was just copied from organization feature for now.